src/pQueue.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+export default class pQueue {
```

Comment:
I left this note above, but I'll repeat here: consider renaming this class to `PriorityQueue`.  It is best practice to use capital camel case for type (class) names, and lower camel case for variable names.